src/main/webapp/babel-src/mainPage.js

```diff
@@ -425,20 +441,22 @@ class MainPage extends React.Component {
         let sourceUrl = this.state.courseSequenceObject.sourceUrl;
         let minTotalCredits = this.state.courseSequenceObject.minTotalCredits;
         let sequenceInfo = this.state.courseSequenceObject.sequenceInfo;
-        let programPrettyName = (sequenceInfo && !this.state.courseSequenceObject.isLoading) ? generatePrettyProgramName(sequenceInfo.program, sequenceInfo.option, sequenceInfo.entryType, minTotalCredits)
-                                                                           : "";
+        let programPrettyName = (sequenceInfo && !this.state.courseSequenceObject.isLoading) ? generatePrettyProgramName(sequenceInfo.program, sequenceInfo.option, sequenceInfo.entryType, minTotalCredits) : "";
+        let siteName = <div style={{display: 'inline-block'}}>{UI_STRINGS.SITE_NAME}</div>;
```

Comment:
create inline style constant in util.js, or apply className to siteName div and add the css in index.css (option two is better if you can figure it out)

Reply:
Done (option 2)